framework/helpers/channels.go

```diff
@@ -33,6 +33,7 @@ func TryReceive[V any](ch <-chan V, timeout time.Duration) opt.Maybe[V] {
 // RequireValue tries to receive a value and returns it if successful, or causes the test
 // to fail and terminate immediately if it timed out.
 func RequireValue[V any](t TestContext, ch <-chan V, timeout time.Duration) V {
+	t.Helper()
```

Comment:
Doing this just improves the stacktraces for failures, so the failure line points to whoever called this method.